internal/pkg/template/templates/workloads/partials/cf/state-machine-definition.json.yml

```diff
@@ -13,7 +13,12 @@
       "Resource": "arn:${Partition}:states:::ecs:runTask.sync",
       "Parameters": {
         "LaunchType": "FARGATE",
-        "PlatformVersion": "1.4.0",
+        "PlatformVersion":
+          {{- if eq .Platform.OS "WINDOWS_SERVER_2019_CORE" "WINDOWS_SERVER_2019_FULL"}}
+            "1.0.0",
```

Comment:
Wut? Is it because Fargate windows is only planning to supports `v1.0.0`?

Reply:
Yes, that's what I was told.

Reply:
I had no idea that 1.0.0 was even a valid option anymore!

Reply:
But what about the other features that might require later version? For example, if we make it `v1.0.0` then will CFN  fail when `exec` is on?

Reply:
"luckily," `exec` will not be supported

Reply:
AND these versions correspond to each platform family!

Reply:
Yeah but if the users they don't remove `exec: on`, does CFN throw any error? I wonder if we should do something such as erroring out earlier...

Reply:
Sorry I was unclear earlier.
OSFamily is indicated in the Task Definition. Platform Family is inherited from OS Family. And the Platform Version refers to just that Platform Family.

Reply:
Yup, the next PR removes `exec: on` and validates that they don't add it if it's Windows

Reply:
and also checks that EFS (also unsupported) isn't enabled